provider/swarm.go

```diff
@@ -39,7 +39,10 @@ func (p SwarmProvider) Name() string {
 
 func (p SwarmProvider) Scale(serviceId string, target int, direction bool) error {
 	ctx := context.Background()
-	service, _, err := p.dockerClient.ServiceInspectWithRaw(ctx, serviceId)
+	// Correct bug #41 using docker/docker v17.06.1-ce-rc4
+	siopts := types.ServiceInspectOptions{}
+	siopts.InsertDefaults = true
```

Comment:
I would avoid instantiating and then assigning `siopts` here.
It's just not needed anywhere else, it even fits 120 columns!